wherehows-web/app/components/dataset-aclaccess-users.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
```

Comment:
All new JS source files in the project should be TypeScript only, please update this as well as other files in this review